next/components/Atoms/DateCardDispaly.tsx

```diff
@@ -11,14 +12,46 @@ interface DateCardProps {
 }
 
 const DateCardDisplay = ({ dateFrom, dateTo, textSize, wrapperClass }: DateCardProps) => {
```

Comment:
There is typo in file name "dispaly" (the const is okay :) I just put the comment here)

Reply:
Fixed in next commit.